lib/pure/parsecfg.nim

```diff
@@ -306,6 +306,7 @@ proc getSymbol(c: var CfgParser, tok: var Token) =
     add(tok.literal, c.buf[pos])
     inc(pos)
     if not (c.buf[pos] in SymChars): break
+  tok.literal = tok.literal.strip
```

Comment:
no strong opinion here; is that so that
`[  No Values  ]` behaves as `[No Values]` ?

Reply:
`key` and `value` will ignore space around in Python, I don't know whether `section` should.
https://docs.python.org/3/library/configparser.html#supported-ini-file-structure

Reply:
according to https://en.wikipedia.org/wiki/INI_file it varies:

> Interpretation of whitespace varies. Most implementations ignore leading and trailing whitespace around the outside of the property name. Some even ignore whitespace within values (for example, making "host name" and "hostname" equivalent). Some implementations also ignore leading and trailing whitespace around the property value; others consider all characters following the equals sign (including whitespace) to be part of the value.

then, how about:
```nim
type IniOptions* = enum iniKeepWhitespace # etc
proc initCfgParser*(options: set[IniOptions] = {}): CfgParser = ...
```

Reply:
@xflywind before implementing this, please wait until somebody else agrees on the proposed solution.

Reply:
The same behaviour as before

```nim
import parsecfg, streams
var dict = loadConfig(newStringStream("""[ Simple_Values ]
    key  =  value  """))

echo dict
```

[Simple_Values]
key=value

Reply:
@narimiran 
Yeah, I won't implement that. I just keep the same behaviour as before.

Feature PR will be postponed at next PR, this PR is intended to fix the issue.